charts/api/values.yaml

```diff
@@ -152,6 +152,11 @@ app:
     tracingEnabled: false
     errorReportingEnabled: false
 
+trustedProxy:
+  # if planning to use this, pass a list of proxy IPs or CIDRs:
```

Comment:
I assume it works but I was surprised not to see cidr when grepping the docs. Is that just undocumented functionality?

Reply:
If I understand it correctly, this is basically https://github.com/fideloper/TrustedProxy vendored into Laravel itself, which also does not mention CIDRs, but uses them in its documentation https://github.com/fideloper/TrustedProxy/wiki/IP-Addresses-of-Popular-Services